src/sentry/remote_config/docs/api.md

```diff
@@ -53,105 +39,231 @@ Retrieve the project's configuration.
       "options": {
         "sample_rate": 1.0,
         "traces_sample_rate": 0.5
+      },
+      "version": 1
+    }
+    ```
+
+## Configurations [/projects/<organization_id_or_slug>/<project_id_or_slug>/configurations/]
+
+| Field              | Type             | Description                                                                   |
+| ------------------ | ---------------- | ----------------------------------------------------------------------------- |
+| environment        | optional[string] | The environment the configuration is associated with.                         |
+| id                 | string           | A server generted unique identifier.                                          |
+| name               | optional[string] | A custom name distinguishing the configuration from the default project name. |
+| sample_rate        | number           |                                                                               |
+| traces_sample_rate | number           |                                                                               |
+| version            | number           | The current configuration version. Initialized to 1.                          |
+
+### Get Configurations [GET]
+
+Retrieve configurations.
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "environment": null,
+        "id": "0b88ac27a7b444a6baeb312c0493aed5",
+        "name": null,
+        "sample_rate": 0,
+        "traces_sample_rate": 0.75,
+        "version": 1
       }
+    ]
+  }
+  ```
+
+### Create Configuration [POST]
+
+Create a new configuration. The version field is ignored on create. Version is initialized to 1.
```

Comment:
is `version` an auto-increment whenever a change is made? or is it like a schema-version?

Reply:
Its manually incremented by user update.  Its to prevent concurrent writes.  The integer version can be associated with errors/traces/whatever and act as a version identifier but that's not totally its intent.

Reply:
Its to "help" prevent concurrent writes.  This doesn't actually prevent it without a lock.  I need to think more about how to handle concurrent publishes.